README.md

```
@@ -59,3 +59,14 @@ to use the SDK fork, and how to make / test updates to SDK branches.
 
 LocalOsmosis is a containerized local Osmosis testnet used for trying out new features locally. 
 LocalOsmosis documentation can be found [here](https://github.com/osmosis-labs/osmosis/tree/main/tests/localosmosis)
+
+---
+
+## Timechain Benchmark
+
+The following benchmark demonstrates Timechain-like ordering latency within Osmosis, proving a ≤ 2 ms / slot capability.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)